src/dosbox.cpp

```diff
@@ -444,18 +444,18 @@ void DOSBOX_Init()
 	secprop = control->AddSection_prop("dosbox", &DOSBOX_RealInit);
 	pstring = secprop->Add_string("language", always, "");
 	pstring->Set_help(
-	        "Select a language to use: de, en, es, fr, it, nl, pl, and ru\n"
+	        "Select a language to use: de, en, es, fr, it, nl, pl, and ru (unset by default)\n"
 	        "Notes: This setting will override the 'LANG' environment, if set.\n"
 	        "       The 'resources/translations' directory bundled with the executable holds\n"
 	        "       these files. Please keep it along-side the executable to support this\n"
 	        "       feature.");
 	pstring = secprop->Add_string("machine", only_at_start, "svga_s3");
 	pstring->Set_values(machines);
-	pstring->Set_help("The type of machine DOSBox tries to emulate.");
+	pstring->Set_help("The type of machine DOSBox tries to emulate ('svga_s3' by default).");
 
 	pstring = secprop->Add_path("captures", always, "capture");
 	pstring->Set_help(
-	        "Directory where things like wave, midi, screenshot get captured.");
+	        "Directory where things like wave, midi, screenshot get captured ('capture' by default).");
```

Comment:
Line over 79 characters - I suggest to re-wrap descriptions like this, otherwise command:

`config -h dosbox captures`

will produce broken help.

Reply:
Very good point; I did not bother re-wrapping everything to 79 chars because I only looked at the generated config written by `config -wc blah`. Kinda forgot that these descriptions can be printed to a 80-column DOS screen as well.

I'll just re-wrap everything in a second commit.